File: Chapter 1/(c)/c.java
import java.util.Scanner;
class c
{
    static public void main(String args[])
    {
        float sum=0;
        float []sub=new float[5];
        Scanner input=new Scanner(System.in);

        for(int i=0;i<sub.length;++i)
        {
                System.out.print("Enter the marks : ");
                sub[i]=input.nextFloat();
                sum=sum+sub[i];

        }
        System.out.println("Average of marks is "+ sum/5);
        System.out.println("Percentage is "+ sum/5 +" %");
    }
}

File: Chapter 1/(c)/c.c
#include <stdio.h>

int main(int argc, char *args[])
{
	float sub[5], sum;
	sum = 0;
	for (int i = 0; i < 5; ++i)
	{
		printf("Enter the marks ", sub[i]);
		scanf("%f", &sub[i]);
		sum = sum + sub[i];
	}

	printf("Average of marks %0.3f ", sum / 5);
	printf("Percentage = %0.3f %%", sum / 5);

	return 0;
}